testing/mock.ts

```diff
@@ -1082,15 +1097,15 @@ export function stub<
   func?: (this: Self, ...args: Args) => Return,
 ): Stub<Self, Args, Return> {
   if (self[property] !== undefined && typeof self[property] !== "function") {
-    throw new MockError("Property is not an instance method");
+    throw new TypeError("Property is not an instance method");
```

Comment:
Ditto. This looks like an error on attempting invalid stubbing. Why isn't this MockError?